scripts/mps-generate-schemas.sh

```diff
@@ -18,16 +18,18 @@ outdir=${1:-"avro"}
 if [[ -d $outdir ]]; then
     rm -r $outdir
 fi
+shift;
+
 mkdir $outdir
 
 total=0
 failed=0
 for schema in $schemas; do
     namespace=$(basename $(dirname $(dirname $schema)))
-    schema_filename=$(basename $schema | sed 's/schema.json/avro.json/g')
+    schema_filename=$(basename $schema)
```

Comment:
I think this is the point where we need to normalize casing of namespace and doctype names. Or perhaps we just have a special case to modify `untrustedModules` to `untrusted_modules`, and we create an issue in mps to fail a test when adding a new schema with capital letters in the name.

Reply:
I think the renaming needs to happen [here](https://github.com/mozilla/mozilla-schema-generator/blob/master/bin/schema_generator.sh#L148) as well since these scripts are AFAIK unused by MSG.

Having MPS fail a test when adding other pings with this property sounds swell.

Reply:
If this script is just used for testing within this repo, then we don't need to worry about snake-casing here. I'd agree that it's the script in MSG where the change needs to happen.

Reply:
Filed https://github.com/mozilla-services/mozilla-pipeline-schemas/issues/355

Reply:
Yep, this script is the prototype of the script in [`gcp-ingestion/ingestion-beam`](https://github.com/mozilla/gcp-ingestion/blob/master/ingestion-beam/bin/generate-avro-schemas) and is mostly for end-to-end testing of encoding json into avro, then importing into bigquery and reading the schema back. The transpiler reads from a file (or stdin), so it doesn't have a bias on how the schema repo is organized.

I'm going to leave this as is, for conciseness.

Reply:
I am going to prep a PR for MGS

Reply:
See https://github.com/mozilla/mozilla-schema-generator/pull/46